nextflow.config

```diff
@@ -6,56 +6,17 @@ manifest {
     mainScript = 'main.nf'
 }
 
+docker.enabled = true
```

Comment:
@Mxrcon , let's replace the `labbactfiocruz` with the name `docker` and then add this line within that file itself.

Also, we should add another profile called `conda` since users can directly use conda with the `mtbseq-nf`  workflow.

Reply:
Yes sure! i was just going to ask you why docker is inside it. So lets remove the docker containers from the standard profile and only keep cpus and memory? and then create conda and docker with instructions about conda recipe / docker image?

Reply:
Agreed 👍

Reply:
Ok, I'll implement it and them ask for a review!